### ISSUE TITLE
Bug: `parameter_type_twostate` doesn't catch 4-state type when `parameter` keyword is omitted.

### ISSUE THREAD

**Issue by DaveMcEwan** [dalance/svlint#206]
When `parameter` keyword is omitted, subsequent parameters are in a `list_of_param_assignments`.
```systemverilog
module M
  #(int A = 0
  , integer B = 0
  ) ();
endmodule
```
- Current behavior: Check only the first param assignment.
- Intended behavior: Check all param assignments in the list.
- Likely similar bugs in `parameter_explicit_type`, `localparam_type_twostate`, and `localparam_explicit_type`.
- See `parameter_default_value` for list-processing logic.
